region/brinstar/pink/Big Pink.json

```diff
@@ -3019,160 +3524,408 @@
+        "Either manipulate it until it can escape, get it stuck swooping in the top right corner of the stairwell then kill it with Bombs,",
+        "or avoid it entirely, and go up through the crumble blocks to the left after manipulating the Sidehoppers to be off-camera at the right of their pit.",
+        "Dodge or kill the Sidehoppers then go through the crumble block before exiting G-mode."
+      ]
+    },
+    {
+      "link": [17, 4],
+      "name": "G-mode Morph, Power Bomb the Blocks",
+      "requires": [
+        "canEnterGMode",
+        {"or": [
+          "Morph",
+          "h_canArtificialMorphIBJ",
+          {"and": [
+            "HiJump",
+            "h_canArtificialMorphSpringBall"
+          ]},
+          {"and": [
+            {"or": [
+              "h_canArtificialMorphSpringBallBombJump",
+              "h_canArtificialMorphSpringFling"
+            ]},
+            {"or": [
+              "h_canArtificialMorphSpringBallBombJump",
+              "canTrickyJump"
+            ]}
+          ]}
+        ]},
+        {"or": [
+          "Morph",
+          "canInsaneJump",
+          {"and": [
+            "canTrickyDodgeEnemies",
+            "h_canArtificialMorphSpringBall"
+          ]},
+          "h_canArtificialMorphPowerBomb",
+          {"enemyDamage": {"enemy": "Reo", "type": "contact", "hits": 1}}
+        ]},
         "h_canArtificialMorphPowerBomb"
       ],
-      "clearsObstacles": ["B"],
+      "collectsItems": [11],
       "flashSuitChecked": true,
       "note": [
-        "Use a Power Bomb to kill any Sidehoppers in the pit below, then roll off and use a second, while exiting G-Mode to destroy the blocks in front of the door.",
-        "Place the Power Bomb far enough left to hit the blocks, but not so far as to lure the Sidehopper.",
-        "This is when Samus is on the right-most squiggly plant, or when there is one tile of air between the right wall and the edge of the screen."
+        "Fully overload PLMs with the item before climbing the room.",
+        "The hardest part is getting past the Reo without Power Bombs, if it is stuck in the stairwell just above the door.",
+        "Either manipulate it until it can escape, get it stuck swooping in the top right corner of the stairwell then kill it with Bombs,",
+        "or avoid it entirely, and go up through the crumble blocks to the left after manipulating the Sidehoppers to be off-camera at the right of their pit.",
+        "Dodge or kill the Sidehoppers then go through the crumble block. Place a Power Bomb then quickly exit G-mode before it explodes to break the Power Bomb blocks."
       ]
+    },
+    {
+      "link": [17, 8],
+      "name": "G-mode Morph",
+      "requires": [
+        "canEnterGMode",
+        "h_canArtificialMorphMovement"
+      ],
+      "collectsItems": [11],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [17, 12],
+      "name": "G-mode Morph",
+      "requires": [
+        "canEnterGMode"
```

Comment:
This is unsound.  PLMs need to be overloaded and you need the item present to use this strat to accomplish that.
So i guess you have add back the `artificialMorphMovement` anyway, but it can be OR'd against `lossOfAccess`.

That's how it plays out in my seeds anyway 😿 I get the item and then need to GMode down to charge.

Reply:
Oh whoops. Glad you caught that. 
`artificialMorphMovement` wouldn't be quite enough without a spring fling, but it wouldn't be too bad. But, anyways, using the camera scroll block will use a different strat.